generative-ai/snippets/test/sendMultiModalPromptWithImage.test.js

```diff
@@ -37,6 +37,6 @@ describe('Generative AI Stream MultiModal with Image', () => {
       `node ./sendMultiModalPromptWithImage.js ${projectId} ${location} ${model}`
     );
     // Ensure that the conversation is what we expect for these images
-    assert(output.match(/city: Rio de Janeiro, Landmark: Christ the Redeemer/));
```

Comment:
![Screenshot from 2024-09-30 13-14-53](https://github.com/user-attachments/assets/b0e3ad29-845d-428e-ad00-8f31b0d72d77)
Reason for change: Those assertions sometimes caused errors.